sdk/identity/identity/test/public/node/clientCertificateCredential.spec.ts

```diff
@@ -73,7 +64,7 @@ describe("ClientCertificateCredential", function () {
 
   it("authenticates with sendCertificateChain", async function (this: Context) {
     if (isLiveMode()) {
-      // Live test run not supported on CI at the moment. Locally should work though.
+      // Running this live causes "AADSTS700030: Invalid certificate - subject name in certificate is not authorized. SubjectNames/SubjectAlternativeNames (up to 10) in token certificate are: identitytestdomain.com"
```

Comment:
For this test to pass, we need to use the SNI certificate, but at the moment it’s only provided in PFX form. We currently don’t have PFX support.

Reply:
Scott Schaab did some changes to the Key Vault, and seems like this test can be added again: https://github.com/Azure/azure-sdk-for-js/pull/20729/commits/aa5fca3de4ccff70bc49c294f4c54c012969e396